cnf-tests/testsuites/pkg/machineconfigpool/machineconfigpool.go

```diff
@@ -135,3 +144,128 @@ func DecodeMCYaml(mcyaml string) (*mcov1.MachineConfig, error) {
 
 	return mc, err
 }
+
+// ApplyKubeleConfigToNode creates a KubeletConfig, a MachineConfigPool and a
+// `node-role.kubernetes.io/<name>` label in order to target a single node in the cluster.
+// The role label is applied to the target node after removing any provious `node-role.kubernetes.io/` label,
+// as MachineConfigOperator doesn't support multiple roles.
```

Comment:
Can you please add to this comment that the returned function is a cleanup that reverses the node label change?